guides/source/active_storage_overview.md

```diff
@@ -518,10 +518,11 @@ To show the uploaded files in a form:
 addEventListener("direct-upload:initialize", event => {
   const { target, detail } = event
   const { id, file } = detail
+  var fileName = $('<span />').text(file.name).html();
```

Comment:
Seems like this is the main part that requires jQuery? If you replace this with some of the suggestions that javan gave it would be vanilla worthy 🎊